code/modules/martial_arts/judo.dm

```diff
@@ -0,0 +1,53 @@
+/datum/martial_art/judo
+	name = "Corporate Judo"
+	has_explaination_verb = TRUE
+	no_baton = TRUE
+	combos = list(/datum/martial_combo/judo/discombobulate, /datum/martial_combo/judo/eyepoke, /datum/martial_combo/judo/judothrow, /datum/martial_combo/judo/armbar, /datum/martial_combo/judo/fatailty)
+	var/armbar_active = FALSE //used to check if the victim is in an armbar, useful for finishing moves
+
+//Corporate Judo Belt
+
+/obj/item/storage/belt/champion/judo
+	name = "Corporate Judo Belt"
+	var/datum/martial_art/judo/style = new
+
+/obj/item/storage/belt/champion/judo/equipped(mob/user, slot)
+	if(!ishuman(user))
+		return
+	if(slot == slot_belt)
+		var/mob/living/carbon/human/H = user
+		if(HAS_TRAIT(user, TRAIT_PACIFISM))
+			to_chat(H, "<span class='warning'>The arts of Corporate Judo echo uselessly in your head, the thought of their violence repulsive to you!</span>")
+			return
+		style.teach(H,1)
+		to_chat(H, "<span class = 'userdanger'>The belt's nanites infuse you with the prowess of a black belt in Corporate Judo!</span>")
+		to_chat(H, "<span class = 'danger'>See the martial arts tab for an explantion of combos!.</span>")
```

Comment:
```suggestion
		to_chat(H, "<span class = 'danger'>See the martial arts tab for an explantion of combos.</span>")
```

Reply:
whoops.

Reply:
It is..still there?

Reply:
Also, "explanation" typo there

Reply:
I think I forgot to save this fix in first pass. Should be good in latest.